### PR TITLE
[SMALLFIX] Update version to 1.9.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by aaudiber**

### CODE REVIEW

File: docs/_config.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ ALLUXIO_MASTER_VERSION_SHORT: master
 ALLUXIO_MAJOR_VERSION: master
 # We must inline the version string (e.g., "1.4.0-SNAPSHOT") rather than using the macro of Alluxio version.
 # Otherwise the macro name remains in the output.
-ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-1.8.0-SNAPSHOT-client.jar
+ALLUXIO_CLIENT_JAR_PATH: /<PATH_TO_ALLUXIO>/client/alluxio-1.9.0-SNAPSHOT-client.jar
```
Comment (apc999) [inline docs/_config.yml]: update line 9 to `1.8.0` too?

File: README.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 Alluxio (formerly Tachyon)
 =======
 
-The master branch is in version 1.8.0-SNAPSHOT:
+The master branch is in version 1.9.0-SNAPSHOT:
```
Comment (calvinjia) [inline README.md]: Should we also update the stable release versions below?

Comment (aaudiber) [reply README.md]: Not until 1.8.0 is out the door

File: docs/_config.yml (hunk 1 above)
Comment (aaudiber) [reply docs/_config.yml]: I will update it when 1.8.0 is released and available for download

**Comment by aaudiber** [comment]
Jenkins, test this please

**Pull request merged**